src/fromager/overrides.py

```diff
@@ -98,6 +98,7 @@ def patches_for_requirement(
     patches_dir: pathlib.Path,
     req: Requirement,
     version: Version,
+    variant: str = "",
```

Comment:
The variant name should be passed by passing a `Context` object as first argument.

Reply:
That would be the only method with a `Context` object in this package. This is mostly helpers. And an extra variable with a default value reduces the size of the change.

Not saying I'm against the idea. Just giving more context about why I did it this way.